utils/src/main/java/software/amazon/awssdk/utils/StringUtils.java

```diff
@@ -603,4 +620,142 @@ public static String uncapitalize(final String str) {
+     *
+     * @param ch the character to be tested
+     * @return true if the character is white  space, false otherwise.
+     */
+    private static boolean isWhiteSpace(final char ch) {
+        if (ch == CHAR_SPACE) {
+            return true;
+        }
+        if (ch == CHAR_TAB) {
+            return true;
+        }
+        if (ch == CHAR_NEW_LINE) {
+            return true;
+        }
+        if (ch == CHAR_VERTICAL_TAB) {
+            return true;
+        }
+        if (ch == CHAR_CARRIAGE_RETURN) {
+            return true;
+        }
+        if (ch == CHAR_FORM_FEED) {
+            return true;
+        }
+        return false;
+    }
+
+    public static String fromInteger(Integer value) {
+        return Integer.toString(value);
+    }
+
+    public static String fromLong(Long value) {
+        return Long.toString(value);
+    }
+
+    public static String fromString(String value) {
+        return value;
+    }
+
+    public static String fromBoolean(Boolean value) {
+        return Boolean.toString(value);
+    }
+
+    public static String fromFloat(Float value) {
+        return Float.toString(value);
+    }
+
+    /**
+     * Converts the specified instant to an ISO 8601 timestamp string and
+     * returns it.
+     *
+     * @param value
+     *            The instant to format as an ISO 8601 timestamp string.
+     *
+     * @return An ISO 8601 timestamp string created from the specified instant.
+     */
+    public static String fromInstant(Instant value) {
+        return DateUtils.formatIso8601Date(value);
+    }
+
+    /**
+     * Returns the string representation of the specified double.
+     *
+     * @param d
+     *            The double to represent as a string.
+     *
+     * @return The string representation of the specified double.
+     */
+    public static String fromDouble(Double d) {
+        return Double.toString(d);
+    }
+
+    /**
+     * Returns the string representation of the specified Byte.
+     *
+     * @param b
+     *            The Byte to represent as a string.
+     *
+     * @return The string representation of the specified Byte.
+     */
+    public static String fromByte(Byte b) {
+        return Byte.toString(b);
+    }
+
+    /**
+     * Base64 encodes the data in the specified byte buffer (from the current
+     * position to the buffer's limit) and returns it as a base64 encoded
+     * string.
+     *
+     * @param byteBuffer
+     *            The data to base64 encode and return as a string; must not be
+     *            null.
+     *
+     * @return The base64 encoded contents of the specified byte buffer.
+     */
+    public static String fromByteBuffer(ByteBuffer byteBuffer) {
+        return Base64Utils.encodeAsString(copyBytesFrom(byteBuffer));
+    }
+
+    /**
+     * @return true if the given value is either null or the empty string
+     */
+    public static boolean isNullOrEmpty(String value) {
+        return value == null || value.isEmpty();
+    }
```

Comment:
This is the same as `isBlank`. Can we just use that instead?

Reply:
ok. I removed StringUtils.isNullOrEmpty